ui/app/templates/components/job-page/parts/title.hbs

```diff
@@ -42,7 +53,13 @@
         @confirmText="Yes, Start Job"
         @confirmationMessage="Are you sure you want to start this job?"
         @awaitingConfirmation={{this.startJob.isRunning}}
-        @onConfirm={{perform this.startJob}} />
+        @onConfirm={{perform this.startJob}}
+        {{keyboard-shortcut 
+          label="Exec"
+          pattern=(array "s" "t" "a" "r" "t")
+          action=(perform this.startJob)
+        }}
```

Comment:
Would it be possible to add some kind of toast confirmation on some of these? They happen rather quickly and it may be hard to tell if they were triggered by a shortcut or something that happened in the backend at the same time.

Reply:
This is a great suggestion, yeah.